app/assets/javascripts/components/todos/main.coffee

```diff
@@ -24,7 +24,7 @@ Constants = @Todos.Constants
       </div>
       <div className='Todos-container'>
         <NewTodo ref='newTodo' onCreateTask={@onCreateTask} />
-        <TodoList tasks={@state.tasks} onChangeTask={@onChangeTask} />
+        <TodoList tasks={@state.tasks} onChangeTask={@onChangeTask} deleteTask={@deleteTask}/>
```

Comment:
Can you change the name to `onDeleteTask` from `deleteTask` ? Since `@deleteTask` is the callback function, so I would like to keep the same naming rule as other properties like `onChangeTask` .